mkdocs.yml

```
@@ -2,14 +2,15 @@ docs_dir: docs/book
 site_dir: docs/html
 nav:
     - Home: index.md
-    - Introduction: intro.md
-    - Installation: installation.md
-    - Usage: usage.md
-    - Reference:
-        - Configuration: configuration.md
-        - "Advanced Usage": advanced.md
-    - 'Application Integration':
-        - 'Stand-Alone': application-integration/stand-alone.md
+    - v2:
+        - Introduction: v2/intro.md
+        - Installation: v2/installation.md
+        - Usage: v2/usage.md
+        - Reference:
+            - Configuration: v2/configuration.md
+            - "Advanced Usage": v2/advanced.md
```

The old "reference" entry can always be deleted. This allows for better nesting in the future.

```suggestion
        - Configuration: v2/configuration.md
        - "Advanced Usage": v2/advanced.md
```

---

I'll try to remember that next time :)